apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabaseBase.cs

```diff
@@ -113,15 +110,9 @@ protected SpannerTestDatabaseBase(string projectId, EmulatorDetection emulatorDe
         // In production, we expect the instance to already exist.
         MaybeCreateInstanceOnEmulator(projectId);
 
-        if (Fresh)
-        {
-            CreateDatabase();
-            Logger.DefaultLogger.Debug($"Created database {SpannerDatabase}.");
-        }
-        else
-        {
-            Logger.DefaultLogger.Debug($"Using existing database {SpannerDatabase}");
-        }
+        Fresh = TryCreateDatabase();
+        Logger.DefaultLogger.Debug($"Created database {SpannerDatabase}.");
```

Comment:
This message isn't correct now. If database is created fresh, it should be "created message" else "using existing database" message,

Reply:
Also, I see we provide message in the `TryCreateDatabase` methods. We can probably optimize that.

Reply:
corrected log.